ratelimit/utils.py

```diff
@@ -98,7 +98,7 @@ def _make_cache_key(group, rate, value, methods):
 
 def is_ratelimited(request, group=None, fn=None, key=None, rate=None,
                    method=ALL, increment=False):
-    if not key:
+    if key is None:
```

Comment:
on the other hand, it's OK to drop this check here, since `get_usage_count` will also throw it